libs/datamodel/connectors/sql-datamodel-connector/src/mysql_datamodel_connector.rs

```diff
@@ -291,19 +285,20 @@ impl Connector for MySqlDatamodelConnector {
             MySqlType::LongText => (LONG_TEXT_TYPE_NAME, vec![]),
             MySqlType::Date => (DATE_TYPE_NAME, vec![]),
             MySqlType::Time(x) => match x {
-                Some(arg) => (TIME_TYPE_NAME, vec![arg]),
-                None => (TIME_TYPE_NAME, vec![]),
+                0 => (TIME_TYPE_NAME, vec![]),
+                arg => (TIME_TYPE_NAME, vec![arg]),
             },
             MySqlType::DateTime(x) => match x {
-                Some(arg) => (DATETIME_TYPE_NAME, vec![arg]),
-                None => (DATETIME_TYPE_NAME, vec![]),
+                0 => (DATETIME_TYPE_NAME, vec![]),
+                arg => (DATETIME_TYPE_NAME, vec![arg]),
             },
             MySqlType::Timestamp(x) => match x {
-                Some(arg) => (TIMESTAMP_TYPE_NAME, vec![arg]),
-                None => (TIMESTAMP_TYPE_NAME, vec![]),
+                0 => (TIMESTAMP_TYPE_NAME, vec![]),
+                arg => (TIMESTAMP_TYPE_NAME, vec![arg]),
             },
```

Comment:
this might lead to changes when re-rendering manually written schemas.  timestamp(0) might become timestamp